packages/ts/src/core/container/index.ts

```diff
@@ -62,15 +68,24 @@ export class ContainerCore {
     }
 
     // Apply the `aria-label` attribute
-    select(this._container)
-      .attr('aria-label', config.ariaLabel)
+    let description = ''
+    if (config.ariaLabel) {
+      description = config.ariaLabel
+    } else if (config.ariaAutoLabel) {
```

Comment:
I find this naming a little bit confusing: `config.ariaAutoLabel`. I think I would prefer `config.autoAriaLabel`. Or `config.autoDescription` and `config.description` instead of `config.ariaLabel`.